### PR TITLE
I've finished this task. Can you please check it?

### PR DESCRIPTION
**Pull request by IvanIKS**

### CODE REVIEW

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -3,5 +3,13 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] ballArray = new Ball[3];
+        for (int i = 0; i < ballArray.length; ++i) {
+            ballArray[i] = lottery.getRandomBall();
+        }
+        for (Ball ball : ballArray) {
+            System.out.println(ball.toString());
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Application.java:12]:
```suggestion
            System.out.println(ball);
```

File: src/main/java/core/basesyntax/Color.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Color {
+    Black,
+    White,
+    Red,
+    Blue,
+    Green,
+    Yellow,
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Color.java:9]: uppercase

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 3]
```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/ColorSupplier.java:7]: image

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 4]
```diff
@@ -1,7 +1,12 @@
...
+        int index = random.nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return "" + color;
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/ColorSupplier.java:10]: image

**Review by Rommelua — CHANGES_REQUESTED** [review]
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery
Before you create PR with your homework, you need to go through the checklist under the task and correct all the points described there.

**Comment by IvanIKS** [comment]
I have revorked the code in the way you suggested: used "ball" instead of "ball.toString", changed text to uppercase, used .name() and moved Random to class level. Can you please check the task again?

File: src/main/java/core/basesyntax/Lottery.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        Ball ball = new Ball();
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(random.nextInt(100));
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Lottery.java:11]: image

**Review by Rommelua — CHANGES_REQUESTED** [review]

**Comment by IvanIKS** [comment]
I've changed "100" with variable "maxNumber" that represents the maximal number of ball. Can you please check it?

**Review by sokimaaa — CHANGES_REQUESTED** [review]

**Comment by IvanIKS** [comment]
I'm sorry I did not commited it earlier.

**Review by sokimaaa — APPROVED** [review]

**Pull request closed without merging**